README.rst

```diff
@@ -55,6 +55,17 @@ extendable and language independent. coala is written with a lower case
 
 Read more at our `documentation <http://coala.rtfd.org/>`__.
 
+Why use coala?
+--------------
+
+- coala provides built-in checking routines (named bears in coala).
+- Serves your requirement: You can easily write your own checks (using bears).
+- coala provides user-friendly interfaces like json, formatted and interactive
+  output in the CLI and plugins for various editors are available as well.
+- Optimal Performance: coala manages parallelizing the checking-routines
+  without you having to worry.
+- Unified Interface: One tool to take care of other tools.
```

Comment:
can we rather have "Unified interface: One tool for all programming
languages" or so? As coala isn't only about wrapping others.

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-03-16 13:14 GMT+01:00 priests <notifications@github.com>:

> In README.rst
> <https://github.com/coala-analyzer/coala/pull/1889#discussion_r56323093>:
>
> > @@ -55,6 +55,17 @@ extendable and language independent. coala is written with a lower case
> >
> >  Read more at our `documentation <http://coala.rtfd.org/>`__.
> >
> > +Why use coala?
> > +--------------
> > +
> > +- coala provides built-in checking routines (named bears in coala).
> > +- Serves your requirement: You can easily write your own checks (using bears).
> > +- coala provides user-friendly interfaces like json, formatted and interactive
> > +  output in the CLI and plugins for various editors are available as well.
> > +- Optimal Performance: coala manages parallelizing the checking-routines
> > +  without you having to worry.
> > +- Unified Interface: One tool to take care of other tools.
>
> Yup highlighting the catch phrases is good idea, may be in all items ,
> like :
> [image: screenshot from 2016-03-16 17-43-07]
> <https://cloud.githubusercontent.com/assets/8680142/13811894/a70f0d66-eb9e-11e5-8e01-61fe5c8b3138.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1889/files/575f402bf6dd50d259371bd111449d815d7e373e#r56323093>
>